aws-s3/spring-cloud-starter-stream-source-s3/src/main/java/org/springframework/cloud/stream/app/s3/source/AmazonS3SourceProperties.java

```diff
@@ -26,7 +26,7 @@
 /**
  * @author Artem Bilan
  */
-@ConfigurationProperties
+@ConfigurationProperties("s3")
```

Comment:
This isn't a problem, as now for example it isn't a problem without any prefix.  The stream deployer looks at each part of the stream and will use the appropriate prefix, they are separate deployment requests to the underlying runtime.  I did speak w/ Marius about how this would work if the two apps were composed in SCSt and that looks ok as well with a little bit of work.